.github/workflows/_static-analysis.yml

```diff
@@ -3,6 +3,29 @@ on:
   workflow_call:
 
 jobs:
+  pr-title-check:
+    runs-on: ubuntu-22.04
```

Comment:
I have used https://github.com/marketplace/actions/semantic-pull-request before in other projects and it worked well.

Reply:
👍